### PR TITLE
Feature/fro 78 modify search 2

### PR DESCRIPTION
**Pull request by joel-wright**

### CODE REVIEW

**Review by PatFawbertMills — APPROVED** [review]
LGTM but then again I did write it

File: frontend/src/components/CountryLink.tsx [hunk 1]
```diff
@@ -18,4 +18,4 @@ export const CountryLink: FC<TCountryLink> = ({ countryCode, children }) => {
       <a className="flex underline text-blue-600">{children}</a>
     </Link>
   );
-};
+};
```
Comment (diversemix) [inline frontend/src/components/CountryLink.tsx:21]: not a show stopper - but the linter might bleat

**Review by diversemix — APPROVED** [review]
lgtm - may wish to see the comment re: geographies / countries :+1:

File: frontend/src/pages/index.tsx [hunk 2]
```diff
@@ -29,11 +29,17 @@ const IndexPage = () => {
   when returning to this page from a previous search
   */
   const configQuery: any = useConfig("config");
-  const { data: { geographies: geographies = [], regions: regions = [], countries: countries = [] } = {} } = configQuery;
+  const { data: { regions: regions = [], countries: countries = [] } = {} } = configQuery;
```
[the remark this answers is not in the document]
Comment (PatFawbertMills) [reply frontend/src/pages/index.tsx:32]: Part of Mark's work was refactoring how we store the countries and regions in the state, they are derived as new collections from the geographies

**Pull request merged**